reference/gdscript.rst

```diff
@@ -485,8 +456,65 @@ return value is null.
         print(b)
         return a + b  # return is optional; without it null is returned
 
+A function can ``return`` at any point. The default return value is ``null``.
+
+Referencing Functions
+^^^^^^^^^^^^^^^^^^^^^
+
+To call a function on a *base class* (i.e. one extended in your current class), 
+prepend ``.`` to the function name:
+
+::
+
+    .basefunc()
+
+Remember that default functions like  ``_init``, and most
+notifications such as ``_enter_tree``, ``_exit_tree``, ``_process``,
+``_fixed_process``, etc. are called in all base classes automatically.
+So there is only a need to call the function explicitly when overloading
+them in some way. 
+
+Contrary to Python, functions are *not* first class objects in GDScript. This 
+means that they cannot be treated as objects and stored in variables. It also
+means that they cannot be referenced directly since they are not considered 
+class members (this is for performance reasons).
+
+To reference a function in another class instance, one must instead use 
```

Comment:
`mynode.myfunction(args)` works, `call` or `FuncRef` are only necessary when passing a function as argument or storing it as a variable